### PR TITLE
Email previews

### PR DESCRIPTION
**Pull request by lipis**
[Preview](https://rawgit.com/wireapp/wire-server/emails/services/brig/deb/opt/brig/templates/index.html)

### CODE REVIEW

**Comment by lipis** [comment]
yes yes.. the new emails are not complete and the actual copy needs to be added.

File: services/brig/deb/opt/brig/templates/team/en/email/new-team.html [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+            <tbody style="font-size: 16px;">
+              <tr>
+                <td width="500" colspan="2" style="padding-top: 48px; padding-bottom: 0px;">
+
+                  <h1 style="font-size: 32px; font-weight: 300; text-align: center; line-height: 1.3;">
+                    Your new team
+                  </h1>
+                  <p style="color: #42474A; margin-top: 48px;">
+                    You have just created a team called
+                    <strong>{{teamName}}</strong> on Wire with this email address. Your 30–day free trial starts today.
```
Comment (tiago-loureiro) [inline services/brig/deb/opt/brig/templates/team/en/email/new-team.html]: `${teamName}`
This is meant to be sent _after_ an owner has activated a team account, right?

File: services/brig/deb/opt/brig/templates/team/en/email/new-team.html [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+                      </v:roundrect>
+                    <![endif]-->
+                  </p>
+
+                  <p style="color: #42474A;">
+                    If you have any questions, please visit
+                    <a href="https://support.wire.com/hc/en-us/requests/new" style="color: #018ED6; text-decoration: none;">Wire Support</a>.
+                  </p>
+                  <p style="color: #42474A; margin-bottom: 40px;">
+                    Team ID: {{teamId}}
```
Comment (tiago-loureiro) [inline services/brig/deb/opt/brig/templates/team/en/email/new-team.html]: `${team_id}`

**Comment by lipis** [comment]
If the preview doesn't work.. Cmd+Click or Ctrl+Click it :)

**Comment by lipis** [comment]
So from my side it's kind ready for the first version.. eventually we'll update all the templates to one design, but for now we'll start with the team related emails.

File: services/brig/deb/opt/brig/templates/team/en/email/welcome.html [hunk 3]
```diff
@@ -0,0 +1,121 @@
...
+                        v-text-anchor:middle;
+                        width: 200px;"
+                        arcsize="15%"
+                        stroke="f"
+                        fillcolor="#018ED6"
+                      >
+                      <w:anchorlock/>
+                      <center>
+                    <![endif]-->
+                    <a href="{{url}}" style="
```
Comment (tiago-loureiro) [inline services/brig/deb/opt/brig/templates/team/en/email/welcome.html]: `{{url}}`

File: services/brig/deb/opt/brig/templates/team/en/email/welcome.html [hunk 4]
```diff
@@ -0,0 +1,121 @@
...
+                      </v:roundrect>
+                    <![endif]-->
+                  </p>
+
+                  <p style="color: #42474A;">
+                    If you have any questions, please
+                    <a href="https://support.wire.com/hc/en-us/requests/new" style="color: #018ED6; text-decoration: none;">contact us</a>.
+                  </p>
+                  <p style="color: #42474A; margin-bottom: 40px;">
+                    Team ID: {{team_id}}
```
Comment (tiago-loureiro) [inline services/brig/deb/opt/brig/templates/team/en/email/welcome.html]: `{{team_id}}`

**Comment by tiago-loureiro** [comment]
There are still a few `{{ }}` variables instead of `${ }` which we need for the templating, can we fix those?
Reg. `new-team` and `welcome`: can we perhaps make them a bit more descriptive such as `new-team-welcome` and `new-team_member-welcome` or similar?
Then a little squash and :shipit:

File: services/brig/deb/opt/brig/templates/team/en/email/invitation.txt [hunk 5]
```diff
@@ -14,5 +14,3 @@ Please don't reply to this message.
 
 
 (c) Wire Swiss GmbH
-
-Privacy Policy  |  Report misuse
```
Comment (lipis) [inline services/brig/deb/opt/brig/templates/team/en/email/invitation.txt]: Not sure if we have to remove them.. all add the actual links

```
Privacy Policy - https://wire.com/...
Report misuse - https://....
```

Comment (infotexture) [reply services/brig/deb/opt/brig/templates/team/en/email/invitation.txt]: Discussed with Yorgos, they're not present in several other plain text messages, so it seemed best to remove them entirely in this context.

The linked variants will remain in the HTML versions.

File: services/brig/deb/opt/brig/templates/team/en/email/new-team.html [hunk 6]
```diff
@@ -0,0 +1,123 @@
...
+                        v-text-anchor:middle;
+                        width: 200px;"
+                        arcsize="15%"
+                        stroke="f"
+                        fillcolor="#018ED6"
+                      >
+                      <w:anchorlock/>
+                      <center>
+                    <![endif]-->
+                    <a href="${new-team-manage}" style="
```
Comment (tiago-loureiro) [inline services/brig/deb/opt/brig/templates/team/en/email/new-team.html]: Where is this link meant to point to?

Comment (lipis) [reply services/brig/deb/opt/brig/templates/team/en/email/new-team.html]: @Yorgos-V? It should be https://teams.wire.com/login/ I guess

File: services/brig/deb/opt/brig/templates/team/en/email/welcome.txt [hunk 7]
```diff
@@ -0,0 +1,10 @@
+Welcome to "Team Name".
+
+You have just joined a team called ${team-name} on Wire with ${email}.
+
+Wire combines strong encryption, a rich feature set and ease-of-use in one app like never before. Works on all popular platforms.
+
+${welcome-download}
```
Comment (tiago-loureiro) [inline services/brig/deb/opt/brig/templates/team/en/email/welcome.txt]: And this link?

**Comment by lipis** [comment]
Kaboom!

**Pull request merged**